server/src/ankaios_server.rs

```diff
@@ -987,13 +968,17 @@ mod tests {
         assert!(comm_middle_ware_receiver.try_recv().is_err());
     }
 
+
+    // TODO the following test tests also the workload state db
+    // This should be avoided, the tests should be 
+
```

Comment:
Shall this be done in separate PR or is it missing inside this PR?

Reply:
A separate one
```suggestion
```